views/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-qti-item",
-    "version": "0.3.3",
+    "version": "0.3.4",
```

Comment:
1. Why bump version in package.json if not any library upgrade?
2. If needed, push `package.lock` too after `npm i` in this `views/` folder

Reply:
Good point, you are right